openbb_terminal/econometrics/econometrics_model.py

```diff
@@ -326,6 +329,89 @@ def get_coint_df(
     return pd.DataFrame()
 
 
+@log_start_end(log=logger)
+def get_returns(data: pd.Series):
+    """Calculate returns for the given time series
+
+    Parameters
+    ----------
+    data: pd.Series
+        The data series to calculate returns for
+    """
+    return 100 * data.pct_change().dropna()
+
+
+@log_start_end(log=logger)
+def get_garch(
+    data: pd.Series,
+    p: int = 1,
+    o: int = 0,
+    q: int = 1,
+    mean: str = "constant",
+    horizon: int = 100,
+):
```

Comment:
This should include an example of how to use in the sdk.

Also technically not annualized since we get rid of the sqrt